README.md

```diff
@@ -12,7 +12,7 @@ All following examples feature similar functionality: print current offers, prin
 They demonstrate working connection signing and aim to be as minimal as possible for ease of understanding.
 
 - /bash/example.sh - Bash script
-- /cs - C# sample project
+- /cs/API-v1/ - C# sample project
```

Comment:
v1?

Reply:
I can make it v1/v2 both for python and dotnet. wdyt?